test/helpers/test-container.helper.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  PostgreSqlContainer,
+  StartedPostgreSqlContainer,
+} from '@testcontainers/postgresql';
+import { DB } from 'database/schema/db';
+import {
+  CamelCasePlugin,
+  FileMigrationProvider,
+  Kysely,
+  Migrator,
+  PostgresDialect,
+} from 'kysely';
+import { join } from 'path';
+import { promises as fs } from 'fs';
+import * as path from 'path';
+
+import { Pool } from 'pg';
+
+export class TestContainerHelper {
+  private static instance: TestContainerHelper;
+  private static container: StartedPostgreSqlContainer;
+
+  private constructor() {}
+
+  public static getInstance(): TestContainerHelper {
+    if (!TestContainerHelper.instance) {
+      TestContainerHelper.instance = new TestContainerHelper();
+    }
+    return TestContainerHelper.instance;
+  }
+
+  public async startPostgresContainer(): Promise<StartedPostgreSqlContainer> {
+    if (!TestContainerHelper.container) {
+      TestContainerHelper.container = await new PostgreSqlContainer(
```

Comment:
we can have multiple test containers for the project, so it would be a better approach to name this as `pgInstance`